query/dev/system/src/main/java/org/codice/ddf/admin/query/dev/system/fields/ServiceRefProperty.java

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.query.dev.system.fields;
+
+import java.util.List;
+
+import org.codice.ddf.admin.api.fields.EnumValue;
+import org.codice.ddf.admin.common.fields.base.BaseEnumField;
+
+import com.google.common.collect.ImmutableList;
+
+public class ServiceRefProperty extends BaseEnumField<String> {
```

Comment:
Why did we go with this approach rather than separate classes? It seems like this is describing `Field`s that should be re-used in `ServiceReferenceField`, but as enumerated values instead. Seems odd to me.

Reply:
see earlier comment about BundleProperty